builtin/providers/aws/resource_aws_instance.go

```diff
@@ -129,9 +129,11 @@ func resourceAwsInstance() *schema.Resource {
 			},
 
 			"network_interface_id": &schema.Schema{
-				Type:     schema.TypeString,
-				Computed: true,
-				Optional: true,
+
+				Type:          schema.TypeString,
```

Comment:
According to http://docs.aws.amazon.com/cli/latest/reference/ec2/run-instances.html, you can create/launch with multiple ENIs, not just 1. So, then, this should be a TypeSet or TypeList?